build-extras-onesignal.gradle

```diff
@@ -63,5 +65,5 @@ configurations.all { resolutionStrategy {
 }}
 
 dependencies {
-    compile 'com.github.OutSystems:OneSignal-Android-SDK:3.8.2-OS2'
+    compile 'com.github.usernuno:OneSignal-Android-SDK:3.10.3-OS'
```

Comment:
This is pointing to my personal repository to allow the plugin to be tested. Please review the Android SDK before this one. When that one is approved and merged I will be able to point to the correct repository.